applications/app/views/fragments/newGalleryBody.scala.html

```diff
@@ -0,0 +1,127 @@
+                            </h2>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    }
+    @fragments.contentFooter(gallery, related, "media")
+</div>
+
+@galleryItem(adSlots: Seq[String], adInterval: Int, image: model.ImageAsset, rowNum: Int, imageElement: model.ImageElement) = {
+
+    <li id="img-@rowNum" class="gallery__item js-gallery-item" data-link-name="Gallery item | @rowNum">
+        <figure itemscope itemtype="http://schema.org/ImageObject">
+
+            <div class="gallery__figcaption">
+                @image.caption.map { caption =>
+                    <p class="gallery__caption" itemprop="caption">@Html(caption)</p>
+                }
+                @if(image.displayCredit) {
+                    @image.credit.map { credit =>
+                        <p class="gallery__credit" itemprop="author">@credit</p>
+                    }
+                }
+
+                @fragments.share.blockLevelSharing("img-" + rowNum.toString, gallery.sharelinks.elementShares("img-" + rowNum.toString, image.path), gallery.metadata.contentType, isNewGallery = true)
+            </div>
+
+            <div class="gallery__img-container gallery__img-container@if(image.width >= image.height) {--landscape} else {--portrait}">
+                <picture>
+                    @* IE 9 needs this workaround as per https://scottjehl.github.io/picturefill/ *@
+                        <!--[if IE 9]><video style="display: none;"><![endif]-->
+                    @defining({
+                        GalleryMedia.inline
+                    }) { widths =>
+                        @widths.breakpoints.map { breakpointWidth =>
+                            <source media="(min-width: @breakpointWidth.breakpoint.minWidth.getOrElse("0")px) and (-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi)"
+                            sizes="@breakpointWidth.width"
+                            srcset="@ImgSrc.srcsetForBreakpoint(breakpointWidth, widths.breakpoints, maybePath = None, maybeImageMedia = Some(imageElement.images), hidpi = true)" />
+                            <source media="(min-width: @breakpointWidth.breakpoint.minWidth.getOrElse("0")px)"
+                            sizes="@breakpointWidth.width"
+                            srcset="@ImgSrc.srcsetForBreakpoint(breakpointWidth, widths.breakpoints, maybePath = None, maybeImageMedia = Some(imageElement.images))" />
+                        }
+                    }
+
+                    <!--[if IE 9]></video><![endif]-->
+
+                    <img class="gallery__img gallery__img@if(image.width >= image.height) {--landscape} else {--portrait}"
+                    itemprop="contentUrl"
+                    alt="@image.altText.getOrElse("")"
+                    src="@ImgSrc.getFallbackUrl(imageElement.images)" />
+                </picture>
```

Comment:
Will it be possible to re-use `img.scala.html` for the `<picture>` element, like all other pages do?